tests/unit/test_runners.py

```
@@ -541,6 +541,103 @@ class LocalMyCommand(Command):
+	def test_workspace_default_profile(self):
+		"""Test that workspace.default_profile is applied to all workspaces."""
+		from secator.utils_test import clear_modules
+		clear_modules()
+
+		from secator.runners import Command
+		from secator.template import TemplateLoader
+		from secator.utils_test import mock_command
+		from unittest.mock import patch
+
+		class LocalMyCommand(Command):
+			input_types = ['slug']
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)